src/metadata/time.rs

```diff
@@ -48,9 +51,9 @@ impl FileTime {
             day,
             month,
             year,
-            hour: -1,
-            minute: -1,
-            second: -1,
+            hour: 0,
```

Comment:
These need to be -1 as -1 denotes to use the current system time.

Reply:
Fixed